app/code/Magento/Contact/view/frontend/web/css/source/_module.less

```diff
@@ -0,0 +1,50 @@
+// /**
+//  * Copyright © Magento, Inc. All rights reserved.
+//  * See COPYING.txt for license details.
+//  */
+
```

Comment:
Please remove empty lines

Reply:
This is not correct, please use /** without aby extra // at the beginning.

Also please remove the empty lines.